wordpress-mobile/WordPress-Android
# Support message and currentEntry in rewind status

Parent #13328

This PR:
(1) Updates build.gradle to use hash for testing FluxC table drop/create for `RewindStatus`
(2) Adds support for "QUEUED" status in `GetRestoreStatusUseCase` and moves the delay to the bottom. The original delay was accounting for the lag in queued vs. progress status, as queued was not yet supported in fluxC. As of now we are not altering the UI for queued.

**To test:**
- Apply this version over current version of app
- Ensure the app still runs and activity log doesn't crash

- Apply this version as a fresh install
- Ensure the app still runs and activity log doesn't crash

**Merge Instructions**
After [FluxC PR ](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1842 )is merged and tagged, this PR will be updated with correct FluxC version and merged to develop.

PR submission checklist:
